src/db/conditions/InConditionBuilder.php

```
@@ -179,88 +213,126 @@ protected function buildSubqueryInCondition($operator, $columns, $values, &$para
+                $column = $column->expression;
+            }
+
+            $quotedColumns[$i] = $this->quoteColumn($column);
+        }
+
         $vss = [];
+
+        $notEqualOperator = $this->getNotEqualOperator();
+
         foreach ($values as $value) {
             $vs = [];
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $vs. Configured minimum length is 3. (undefined)

(ShortVariable)